<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>552</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>350</width>
    <height>275</height>
   </size>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>208</red>
      <green>208</green>
      <blue>208</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
     <color>
      <red>120</red>
      <green>120</green>
      <blue>120</blue>
     </color>
     <color>
      <red>160</red>
      <green>160</green>
      <blue>160</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>240</red>
      <green>240</green>
      <blue>240</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>128</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
    </active>
    <inactive>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>208</red>
      <green>208</green>
      <blue>208</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
     <color>
      <red>120</red>
      <green>120</green>
      <blue>120</blue>
     </color>
     <color>
      <red>160</red>
      <green>160</green>
      <blue>160</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>240</red>
      <green>240</green>
      <blue>240</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>192</red>
      <green>192</green>
      <blue>192</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
    </inactive>
    <disabled>
     <color>
      <red>120</red>
      <green>120</green>
      <blue>120</blue>
     </color>
     <color>
      <red>208</red>
      <green>208</green>
      <blue>208</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
     <color>
      <red>120</red>
      <green>120</green>
      <blue>120</blue>
     </color>
     <color>
      <red>160</red>
      <green>160</green>
      <blue>160</blue>
     </color>
     <color>
      <red>120</red>
      <green>120</green>
      <blue>120</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>120</red>
      <green>120</green>
      <blue>120</blue>
     </color>
     <color>
      <red>240</red>
      <green>240</green>
      <blue>240</blue>
     </color>
     <color>
      <red>240</red>
      <green>240</green>
      <blue>240</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>128</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>247</red>
      <green>247</green>
      <blue>247</blue>
     </color>
    </disabled>
   </palette>
  </property>
  <property name="font" >
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>Configure Settings</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QListWidget" name="lstPages" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>84</width>
       <height>180</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>84</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="contextMenuPolicy" >
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="verticalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="autoScroll" >
      <bool>false</bool>
     </property>
     <property name="iconSize" >
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="movement" >
      <enum>QListView::Static</enum>
     </property>
     <property name="flow" >
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="spacing" >
      <number>9</number>
     </property>
     <property name="viewMode" >
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes" >
      <bool>true</bool>
     </property>
     <property name="currentRow" >
      <number>0</number>
     </property>
     <item>
      <property name="text" >
       <string>General</string>
      </property>
      <property name="icon" >
       <iconset resource="..\res\vidalia.qrc">:/images/config_general.png</iconset>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Advanced</string>
      </property>
      <property name="icon" >
       <iconset resource="..\res\vidalia.qrc">:/images/config_advanced.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="stackPages" >
       <property name="contextMenuPolicy" >
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="currentIndex" >
        <number>1</number>
       </property>
       <widget class="QWidget" name="pageGeneral" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="grpTorLocation" >
           <property name="contextMenuPolicy" >
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="title" >
            <string>Tor Location</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>4</number>
            </property>
            <property name="spacing" >
             <number>3</number>
            </property>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLineEdit" name="lineTorPath" >
                <property name="enabled" >
                 <bool>true</bool>
                </property>
                <property name="contextMenuPolicy" >
                 <enum>Qt::NoContextMenu</enum>
                </property>
                <property name="toolTip" >
                 <string>Path to Tor's executable</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnBrowseTorPath" >
                <property name="enabled" >
                 <bool>true</bool>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="contextMenuPolicy" >
                 <enum>Qt::NoContextMenu</enum>
                </property>
                <property name="toolTip" >
                 <string>Select path to Tor executable</string>
                </property>
                <property name="text" >
                 <string>Browse</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grpStartup" >
           <property name="contextMenuPolicy" >
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="title" >
            <string>Startup Options</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QCheckBox" name="chkRunWithSys" >
              <property name="contextMenuPolicy" >
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="text" >
               <string>Run Vidalia on system start</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkRunTor" >
              <property name="contextMenuPolicy" >
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="text" >
               <string>Start Tor on Vidalia start</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>309</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageAdvanced" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="grpTorConfigFile" >
           <property name="contextMenuPolicy" >
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="title" >
            <string>Tor Configuration File</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>4</number>
            </property>
            <property name="spacing" >
             <number>3</number>
            </property>
            <item>
             <widget class="QLabel" name="lblTorConfig" >
              <property name="font" >
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="text" >
               <string>Vidalia can start Tor using an alternate Tor configuration file (torrc). If you would like to use a specific torrc, specify it below.</string>
              </property>
              <property name="wordWrap" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLineEdit" name="lineTorConfig" >
                <property name="enabled" >
                 <bool>true</bool>
                </property>
                <property name="contextMenuPolicy" >
                 <enum>Qt::NoContextMenu</enum>
                </property>
                <property name="toolTip" >
                 <string>Location of Tor executable</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnBrowseTorConfig" >
                <property name="enabled" >
                 <bool>true</bool>
                </property>
                <property name="minimumSize" >
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="contextMenuPolicy" >
                 <enum>Qt::NoContextMenu</enum>
                </property>
                <property name="toolTip" >
                 <string>Select path to Tor configuration file</string>
                </property>
                <property name="text" >
                 <string>Browse</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grpControlPort" >
           <property name="contextMenuPolicy" >
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="title" >
            <string>Control Port</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label" >
              <property name="font" >
               <font>
                <family>Arial</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
               </font>
              </property>
              <property name="text" >
               <string>The control port is the port Vidalia uses to communicate with Tor. You only need to change this value if you have a service running on your machine that conflicts with the default port of 9051.</string>
              </property>
              <property name="wordWrap" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item>
               <widget class="QLineEdit" name="lineControlPort" >
                <property name="maximumSize" >
                 <size>
                  <width>50</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="maxLength" >
                 <number>5</number>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignRight</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>301</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave" >
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy" >
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="toolTip" >
          <string>Saves current settings</string>
         </property>
         <property name="text" >
          <string>Save Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel" >
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy" >
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="toolTip" >
          <string>Cancels any changes made to settings</string>
         </property>
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../res/vidalia.qrc" />
 </resources>
 <connections/>
</ui>
